src/display-layer.coffee

```diff
@@ -1212,6 +1223,7 @@ class DisplayLayer
     }
 
   getScreenLineCount: ->
+    @recomputeScreenLinesIfNeeded()
```

Comment:
Should we add this to methods like `getRightmostScreenPosition` and `lineLengthForScreenRow` as well?